### DISCUSSION TITLE
How can I fix the currently open directory?

### DISCUSSION THREAD

**Discussion by Asthestarsfalll** [nvim-neo-tree/neo-tree.nvim#1318] [Q&A] [answered]
When I use `nvim` in a directory, I expect to open neo-tree in current directory instead of in parent directroy which contains a git folder.  And when editing files located in the `~` path, such as some configurations in `~/.config`, Neo-Tree will directly open in the `~` path if it's not a git repository. So how can I fix the opening directroy to cwd?

**Comment by cseickel** [comment, marked as answer]
I think you probably have some other plugin that is changing your current working directory. Neo-tree will, by default, be bound to the tab's cwd for a sidebar and the window's cwd for a "current" position tree.

See `:h neo-tree-cwd` for details.
